test/integration/logic/resolvable/UnifyAttributeConcludableTest.java

```diff
@@ -92,15 +94,23 @@ public void setUp() throws IOException {
                                                        "").asDefine());
             tx.commit();
         }
-        rocksTransaction = session.transaction(Arguments.Transaction.Type.READ);
+    }
+
+    @AfterClass
+    public static void tearDown() {
+        session.close();
+        grakn.close();
```

Comment:
also correctly setup and tear down grakn/sessions/transactions in integration tests